### DISCUSSION TITLE
How can I add "Feedback and Ads" sections?

### DISCUSSION THREAD

**Discussion by mdobydullah** [thuliteio/doks#1188] [Q&A] [answered]
Hi Doks team,

Thank you so much for creating such an awesome tool.

I want to add "Feedback and Ads" sections like getdoks. How can I do that?

![Screenshot_19](https://github.com/gethyas/doks/assets/13184472/b02e86ec-effe-4e1b-a503-3469e4a7f923)

Thank you again.

**Comment by h-enk** [comment, marked as answer]
Feedback partial: https://github.com/gethyas/getdoks.org/blob/main/layouts/partials/sidebar/feedback.html
Ads partial: https://github.com/gethyas/getdoks.org/blob/main/layouts/partials/main/carbon-ads.html

Both called from: https://github.com/gethyas/getdoks.org/blob/main/layouts/_default/single.html#L14-L20

**Reply by h-enk** [reply]
Plus some custom styling off course:
- https://github.com/gethyas/getdoks.org/blob/main/assets/scss/common/_custom.scss
- https://github.com/gethyas/getdoks.org/blob/main/assets/scss/components/_ads.scss
